src/modules/app/app.service.ts

```diff
@@ -57,21 +59,48 @@ export class AppService {
     createRoom (roomName: string, isPrivate: boolean) {
         return this.roomRepository.save({
             roomName,
-            isPrivate
+            isPrivate,
+            messages: []
         })
     }
 
-    createMessage (messageContent: string, isImage: boolean, date: Date, userId: string, roomName: string) {
+    async createMessage (messageContent: string, isImage: boolean, date: Date, userId: string, roomName: string) {
```

Comment:
if you have so many parameters you should create one parameter like `message: CreateChatMessage`